sdk/keyvault/Microsoft.Azure.Management.KeyVault/src/Microsoft.Azure.Management.KeyVault.csproj

```diff
@@ -11,11 +11,12 @@
 		</Description>
 		<AssemblyTitle>Microsoft Azure Key Vault Management</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.KeyVault</AssemblyName>
-		<Version>3.1.0-preview.1</Version>
+		<Version>3.1.0-preview.2</Version>
 		<PackageTags>Microsoft Azure key vault management;Key Vault;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
 				- Support CRUD operations to managed HSM
```

Comment:
I didn't remove release notes from last release because I want to keep it until the stable release 3.1.0